modules/oadp-installing-dpa-1-3.adoc

```diff
@@ -108,6 +108,46 @@ spec:
 <15> Specify the name of the `Secret` object that you created. If you do not specify this value, the default name, `{credentials}`, is used. If you specify a custom name, the custom name is used for the snapshot location. If your backup and snapshot locations use different credentials, create separate profiles in the `credentials-velero` file.
 endif::[]
 
+ifdef::installing-oadp-ibm-cloud[]
++
+[source,yaml,subs="attributes+"]
+----
+apiVersion: oadp.openshift.io/v1alpha1
+kind: DataProtectionApplication
+metadata:
+  namespace: openshift-adp
+  name: <dpa_name>
+spec:
+  configuration:
+    velero:
+      defaultPlugins:
+      - openshift
+      - aws
+      - csi
+  backupLocations:
+    - velero:
+        provider: aws # <1>
+        default: true
+        objectStorage:
+          bucket: <bucket_name> # <2>
+          prefix: velero
+        config:
+          insecureSkipTLSVerify: 'true'
+          profile: default
+          region: <region_name> # <3> 
+          s3ForcePathStyle: 'true'
+          s3Url: <s3_url> # <4>
+        credential:
+          key: cloud
+          name: cloud-credentials # <5>
+----
+<1> The provider is `aws` when you use {ibm-cloud-title} as a backup storage location.
+<2> Specify the {ibm-cloud-object-storage} bucket name.
+<3> Specify the COS region name. For example, `eu-gb`.
+<4> Specify the S3 URL of the COS bucket. For example, `http://s3.eu-gb.cloud-object-storage.appdomain.cloud`. Here, `eu-gb` is the region name. Replace the region name according to your bucket region. 
+<5> `cloud-credentials` is the name of the secret you created by using the `access-key` and `secret-access-key` from the `HMAC` credentials.
```

Comment:
```suggestion
<5> Defines the name of the secret you created by using the access key and the secret access key from the `HMAC` credentials.
```

- `cloud-credentials` need not be repeated in the decription.
- `access-key` and `secret-access-key` do not need back ticks.  
https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#inline-code-or-commands